### DISCUSSION TITLE
How well does it perform in terms of supporting Chinese?

### DISCUSSION THREAD

**Discussion by growmuye** [microsoft/LLMLingua#33] [Q&A]
I tried it on https://huggingface.co/spaces/microsoft/LLMLingua, but after compression, the Chinese characters became garbled.

<img width="580" alt="image" src="https://github.com/microsoft/LLMLingua/assets/79145285/d1254eb0-fd56-4b50-b1e0-af809c0bd981">

**Comment by iofu728** [comment]
Hi @growmuye, thank you for your support.

This is regarding the known issue #4; we are working on resolving it as soon as possible.
